common/lib/capa/capa/templates/textline.html

```diff
@@ -1,20 +1,24 @@
 <% doinline = "inline" if inline else "" %>
 
-<div id="inputtype_${id}" class="${'text-input-dynamath' if do_math else ''} capa_inputtype ${doinline} textline" >
-
-    % if preprocessor is not None:
+<div id="inputtype_${id}" class="${'text-input-dynamath' if do_math else ''} capa_inputtype ${doinline} textline">
+% if preprocessor is not None:
     <div class="text-input-dynamath_data ${doinline}" data-preprocessor="${preprocessor['class_name']}"/>
     <div class="script_placeholder" data-src="${preprocessor['script_src']}"/>
-    % endif
+% endif
 
-    % if status in ('unsubmitted', 'correct', 'incorrect', 'partially-correct', 'incomplete'):
+% if status in ('unsubmitted', 'correct', 'incorrect', 'partially-correct', 'incomplete'):
     <div class="${status.classname} ${doinline}" id="status_${id}">
-    % endif
-    % if hidden:
-      <div style="display:none;" name="${hidden}" inputid="input_${id}" />
-    % endif
+% endif
+
+% if hidden:
+    <div style="display:none;" name="${hidden}" inputid="input_${id}" />
+% endif
 
-  <input type="text" name="input_${id}" id="input_${id}" aria-label="${response_data['label']}" ${describedby} value="${value|h}"
+    <label class="question-label" for="input_${id}">${response_data['label']}</label>
```

Comment:
We need to be consistent in using the class for such elements, let's go with `problem-group-label` as suggested by wiki instead of `question-label`.